public/index.html

```diff
@@ -33,7 +33,7 @@
   crossorigin=""
 />
 
-    <title>React App</title>
```

Comment:
it looks like I see the same changes in two different PRs. For consistency, next time please create each branch from develop or send just a one PR